lang/test/org/partiql/lang/eval/EvaluatorTestSuite.kt

```diff
@@ -439,6 +439,52 @@ internal val EVALUATOR_TEST_SUITE: IonResultTestSuite = defineTestSuite {
             """
         )
 
+        test(
```

Comment:
I guess I did add a few tests below--for some reason I cannot recall, however, I see no reason to remove them.